src/app/shared/common-shared.module.ts

```diff
@@ -0,0 +1,44 @@
+import { NgModule } from '@angular/core';
```

Comment:
Good job :+1: @Kajol-Kumari Thats what I was thinking when I was reviewing the previous PR of lazy loading.